sal_interfaces/MTMount/MTMount_Telemetry.xml

```diff
@@ -300,5899 +138,98 @@
-          <Units>unitless</Units>
-          <Count>1</Count>
-      </item>
-     <item>
-          <EFDB_Name>Azimuth_Status_Drive_16</EFDB_Name>
-          <Description>Azimuth_Status_Drive_16</Description>
-          <Size_in_bytes>1</Size_in_bytes>
-          <IDL_Type>string</IDL_Type>
-          <Units>unitless</Units>
-          <Count>1</Count>
-      </item>
-     <item>
-          <EFDB_Name>Azimuth_Current_Drive_1</EFDB_Name>
-          <Description>Azimuth_Current_Drive_1</Description>
-          <IDL_Type>double</IDL_Type>
-          <Units>A</Units>
-          <Count>1</Count>
-      </item>
-     <item>
-          <EFDB_Name>Azimuth_Current_Drive_2</EFDB_Name>
-          <Description>Azimuth_Current_Drive_2</Description>
-          <IDL_Type>double</IDL_Type>
-          <Units>A</Units>
-          <Count>1</Count>
-      </item>
-     <item>
-          <EFDB_Name>Azimuth_Current_Drive_3</EFDB_Name>
-          <Description>Azimuth_Current_Drive_3</Description>
-          <IDL_Type>double</IDL_Type>
-          <Units>A</Units>
-          <Count>1</Count>
-      </item>
-     <item>
-          <EFDB_Name>Azimuth_Current_Drive_4</EFDB_Name>
-          <Description>Azimuth_Current_Drive_4</Description>
-          <IDL_Type>double</IDL_Type>
-          <Units>A</Units>
-          <Count>1</Count>
-      </item>
-     <item>
-          <EFDB_Name>Azimuth_Current_Drive_5</EFDB_Name>
-          <Description>Azimuth_Current_Drive_5</Description>
-          <IDL_Type>double</IDL_Type>
-          <Units>A</Units>
-          <Count>1</Count>
-      </item>
-     <item>
-          <EFDB_Name>Azimuth_Current_Drive_6</EFDB_Name>
-          <Description>Azimuth_Current_Drive_6</Description>
-          <IDL_Type>double</IDL_Type>
-          <Units>A</Units>
-          <Count>1</Count>
-      </item>
-     <item>
-          <EFDB_Name>Azimuth_Current_Drive_7</EFDB_Name>
-          <Description>Azimuth_Current_Drive_7</Description>
-          <IDL_Type>double</IDL_Type>
-          <Units>A</Units>
-          <Count>1</Count>
-      </item>
-     <item>
-          <EFDB_Name>Azimuth_Current_Drive_8</EFDB_Name>
-          <Description>Azimuth_Current_Drive_8</Description>
-          <IDL_Type>double</IDL_Type>
-          <Units>A</Units>
-          <Count>1</Count>
-      </item>
-     <item>
-          <EFDB_Name>Azimuth_Current_Drive_9</EFDB_Name>
-          <Description>Azimuth_Current_Drive_9</Description>
-          <IDL_Type>double</IDL_Type>
-          <Units>A</Units>
-          <Count>1</Count>
-      </item>
+ <EFDB_Topic>MTMount_Safety_System</EFDB_Topic>
      <item>
-          <EFDB_Name>Azimuth_Current_Drive_10</EFDB_Name>
-          <Description>Azimuth_Current_Drive_10</Description>
-          <IDL_Type>double</IDL_Type>
-          <Units>A</Units>
+          <EFDB_Name>CCW_STO</EFDB_Name>
+          <Description>CCW_STO</Description>
+          <IDL_Type>boolean</IDL_Type>
+          <Units>   </Units>
```

Comment:
Units cannot be blank or just whitespace.  Please use `unitless` or `dimensionless`.